### PR TITLE
Make tooltip move after mouse.

### PR DESCRIPTION
**Pull request by ancamaria99**

### CODE REVIEW

File: step-release-vis/src/app/components/environment/environment.ts [hunk 1]
```diff
@@ -105,12 +105,43 @@ export class EnvironmentComponent implements OnInit {
...
+      divTooltip.style.top = (mouseY + 20).toString() + 'px';
+    }
+
+    divTooltip.style.display = 'block';
+  }
+
+  hideTooltip(): void {
+    const tooltip = document.querySelector('app-tooltip');
+    const divTooltip = tooltip.querySelector('div');
+    divTooltip.style.display = 'none';
```
Comment (mracz) [inline step-release-vis/src/app/components/environment/environment.ts:144]: Please add tests for this.

Comment (ancamaria99) [reply step-release-vis/src/app/components/environment/environment.ts:183]: done!

File: step-release-vis/src/app/components/environment/environment_test.ts [hunk 2]
```diff
@@ -72,6 +72,27 @@ describe('EnvironmentComponent', () => {
         expect(component.polygons[i].highlight).toBeFalse();
       }
     });
+
+    it('tooltip should show, move position and disappear', () => {
```
Comment (stofee) [inline step-release-vis/src/app/components/environment/environment_test.ts:76]: fine for now, but usually having a sinlge assert section per test is better practice

**Review by stofee — APPROVED** [review]

File: step-release-vis/src/app/components/environment/environment_test.ts (hunk 2 above)
Comment (mracz) [reply step-release-vis/src/app/components/environment/environment_test.ts:76]: sorry, I have to argue with that. I would like to see this as 3 separate testcases. Setup part can go to a describe's foreach section, so not much of a duplication, but much more cleaner and readable code.

**Pull request merged**